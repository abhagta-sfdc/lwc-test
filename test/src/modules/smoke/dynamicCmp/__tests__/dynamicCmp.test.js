/*
 * Copyright (c) 2018, salesforce.com, inc.
 * All rights reserved.
 * SPDX-License-Identifier: MIT
 * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
 */
import { createElement } from 'lwc';
import DynamicCmp from '../dynamicCmp';

it('should render dynamically imported component', (done) => {
    const element = createElement('smoke-dynamic-import', { is: DynamicCmp });
    document.body.appendChild(element);

    setTimeout(() => {
        expect(element).toMatchSnapshot();
        done();
    });
});
