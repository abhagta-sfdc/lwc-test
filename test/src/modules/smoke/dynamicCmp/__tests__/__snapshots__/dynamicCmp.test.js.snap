// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render dynamically imported component 1`] = `
<smoke-dynamic-import>
  #shadow-root(open)
    <x-test>
      #shadow-root(open)
        <span>
          Child
        </span>
    </x-test>
</smoke-dynamic-import>
`;
